src/components/home/questionnaire-card.tsx

```diff
@@ -27,12 +28,12 @@ const QuestionnaireCard = () => {
     return (
       <CardContent>
         <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
-          {strings.questionnaireCard.questionnaires}
+          {strings.questionnaireProgress?.title}
```

Comment:
This is still using optional syntax- the questionnaireProgress?.title- the ?.

Is there a reason for this, otherwise it should be removed.